core/src/main/scala/org/apache/spark/SparkEnv.scala

```diff
@@ -423,7 +425,6 @@ object SparkEnv extends Logging {
       conf,
       memoryManager,
```

Comment:
there is a comment above that says "     // NB: blockManager is not valid until initialize() is called later."   We may want to update that to mention having to set shuffle manager